impress-org/givewp
# Fix: show correct donation form title on stripe checkout page

Resolves #6377

## Description


I find out that `give_get_donation_form_title ` function which we were using to get the form title returns output for the frontend display (Primarily for a donation receipt). I added a function to the donation repository to return raw donation names from donation meta. 

In legacy logic we were passing only form title as product name:
![Scope Stripe core gateways now use the Gateway API (#6222) · impress-orggivewp@e0b4e4b 2022-04-21 at 11 57 17 PM](https://user-images.githubusercontent.com/1784821/164528537-4834daeb-f01d-488e-a427-dc5f0aa2260b.jpg)


## Affects


It only affects Stripe Checkout (redirect) payment processing.

## Visuals


![give_test 2022-04-21 at 11 56 30 PM](https://user-images.githubusercontent.com/1784821/164528358-b5f521f6-3feb-4564-8a27-c2bf5593e61e.jpg)


## Testing Instructions


You should see the correct product name/ form title on the Stripe checkout page.

## Pre-review Checklist
